command/job_status.go

```diff
@@ -406,6 +406,15 @@ func (c *JobStatusCommand) outputJobInfo(client *api.Client, job *api.Job) error
 		return fmt.Errorf("Error querying latest job deployment: %s", err)
 	}
 
+	jobActions := make([]string, 0)
+	for _, tg := range job.TaskGroups {
+		for _, task := range tg.Tasks {
+			for _, action := range task.Actions {
+				jobActions = append(jobActions, fmt.Sprintf("%s|%s|%s", *tg.Name, task.Name, action.Name))
```

Comment:
```suggestion
			for _, action := range task.Actions {
				jobActions = append(jobActions, map[string]string{
					"group": *tg.Name,
					"task": task.Name,
					"action": action.Name,
				}))
```

(I haven't checked that this is valid code)

with something like this, you wouldn't need to worry about "|"s in the names.

Reply:
Hopefully got a little closer to what you had in mind now!